library/src/main/java/com/novoda/downloadmanager/lib/NotificationCustomiser.java

```diff
@@ -2,7 +2,9 @@
 
 import android.support.v4.app.NotificationCompat;
 
+import com.novoda.downloadmanager.Download;
+
 public interface NotificationCustomiser {
 
-    void modifyQueuedOrDownloadingNotification(NotificationCompat.Builder builder, long batchId);
+    void modifyQueuedOrDownloadingNotification(NotificationCompat.Builder builder, Download download);
```

Comment:
my understanding is that a `Batch` is composed of several downloads. so passing a `Download` object here is a bit confusing isnt it? cause what you really want is the Batch. 
There is a semantic problem I think, cause when we say **a download** it actually means **a video stream and several ads stream** (and this is represented by a **Batch**) 

iam not even sure this makes any sense :)